spec/ttml1.xml

```diff
@@ -4303,6 +4303,12 @@ glyphs do not have an EM square that is distinct from their font's EM square.</p
 <p>If horizontal and vertical sizes are expressed independently, then the
 units of the <loc href="#style-value-length">&lt;length&gt;</loc> values must be the same.</p>
 <p>The <loc href="#style-value-length">&lt;length&gt;</loc> value(s) used to express font size must be non-negative.</p>
+<p><loc href="#style-value-length">&lt;length&gt;</loc> values specified on a <loc href="#layout-vocabulary-region">
```

Comment:
Need to tweak to prevent use of ``<length>`` at start of sentence, e.g., change to "When ``<length>`` values ...".